cli/options/options.go

```diff
@@ -35,9 +35,9 @@ const RPCEndpointFlag = "rpc-endpoint"
 // Network is a set of flags for choosing the network to operate on
 // (privnet/mainnet/testnet).
 var Network = []cli.Flag{
-	cli.BoolFlag{Name: "privnet, p", Usage: "use private network configuration"},
-	cli.BoolFlag{Name: "mainnet, m", Usage: "use mainnet network configuration"},
-	cli.BoolFlag{Name: "testnet, t", Usage: "use testnet network configuration"},
+	cli.BoolFlag{Name: "privnet, p", Usage: "use private network configuration (if --config-path option is specified)"},
+	cli.BoolFlag{Name: "mainnet, m", Usage: "use mainnet network configuration (if --config-path option is specified)"},
+	cli.BoolFlag{Name: "testnet, t", Usage: "use testnet network configuration (if --config-path option is specified)"},
```

Comment:
That's not exactly the case. It's rather `if --config-file is not specified`. I'd expect `neo-go node -m` to still work the same way as before.